### PR TITLE
login: translate to indigo

### PR DESCRIPTION
**Pull request by tylershuster**
fixes https://github.com/urbit/landscape/issues/154

!

@Fang- @matildepark

### CODE REVIEW

File: pkg/arvo/sys/vane/eyre.hoon [hunk 1]
```diff
@@ -236,16 +236,20 @@
...
-  |=  [redirect-url=(unit @t) our=@p]
+  |=  [redirect-url=(unit @t) our=@p failed=?]
   ^-  octs
   =+  redirect-str=?~(redirect-url "" (trip u.redirect-url))
   %-  as-octs:mimes:html
   %-  crip
   %-  en-xml:html
+  =/  favicon  %+
+    weld  "<svg width='10' height='10' viewBox='0 0 10 10' xmlns='http://www.w3.org/2000/svg'>"
+          "<circle r='3.09' cx='5' cy='5' /></svg>"
```
Comment (Fang-) [inline pkg/arvo/sys/vane/eyre.hoon:247]: wait a second, this isn't sail! (^:

(tbc, I don't care all that much.)

File: pkg/arvo/sys/vane/eyre.hoon [hunk 2]
```diff
@@ -254,168 +258,154 @@
...
+          =class  "mono"
+          =required  "true"
+          =minlength  "27"
+          =maxlength  "27"
+          =pattern  "((?:[a-z]\{6}-)\{3}(?:[a-z]\{6}))"
+          =autofocus  "true";
+        ;input(type "hidden", name "redirect", value redirect-str);
+        ;+  ?.  failed
+          ;span;
+          ;span.failed
```
Comment (Fang-) [inline pkg/arvo/sys/vane/eyre.hoon:388]:
```suggestion
        ;+  ?.  failed  ;span;
            ;span.failed
```

File: pkg/arvo/sys/vane/eyre.hoon [hunk 3]
```diff
@@ -254,168 +258,154 @@
...
+    ;script:'''
+            var failSpan = document.querySelector('.failed');
+            if (failSpan) {
+              document.querySelector("input[type=password]").addEventListener('keyup', function (event) {
+                failSpan.style.display = 'none';
+              });
+            }
+            '''
   ==
+::  +render-tan
```
Comment (Fang-) [inline pkg/arvo/sys/vane/eyre.hoon:408]: Looks unintentional.

File: pkg/arvo/sys/vane/eyre.hoon [hunk 4]
```diff
@@ -254,168 +258,154 @@
...
+            Key is incorrect
           ==
-        ==
+        ;button(type "submit"):"Continue"
       ==
     ==
+    ;script:'''
+            var failSpan = document.querySelector('.failed');
+            if (failSpan) {
+              document.querySelector("input[type=password]").addEventListener('keyup', function (event) {
```
Comment (Fang-) [inline pkg/arvo/sys/vane/eyre.hoon:402]: Oof owie my 80 char margins.

File: pkg/arvo/sys/vane/eyre.hoon [hunk 5]
```diff
@@ -254,168 +258,154 @@
...
+          ;span.failed
+            ;svg(xmlns "http://www.w3.org/2000/svg", viewBox "0 0 12 12")
+              ;circle(cx "6", cy "6", r "5.5");
+              ;line(x1 "3.27", y1 "3.27", x2 "8.73", y2 "8.73");
+              ;line(x1 "8.73", y1 "3.27", x2 "3.27", y2 "8.73");
             ==
+            Key is incorrect
           ==
-        ==
+        ;button(type "submit"):"Continue"
```
Comment (Fang-) [inline pkg/arvo/sys/vane/eyre.hoon:396]: I think you want this button at the same indent as the `;span.failed`?

**Review by Fang- — APPROVED** [review]
Just some style nits, this looks fine otherwise. Thank you!

**Comment by matildepark** [comment]
Just amend these commits and we'll merge; you'll want to have them both say `eyre:`

**Comment by matildepark** [comment]
CI fail has to do with GCP permissioning on upload, so this is good to go ...

**Pull request merged**